main/freetype/APKBUILD

```diff
@@ -38,21 +33,16 @@ build() {
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info \
-		--disable-static \
-		|| return 1
-	make || return 1
+		--disable-static
+	make
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install || return 1
+	make DESTDIR="$pkgdir" install
 
 	install -Dm644 "$srcdir"/freetype-profile.sh \
 		"$pkgdir"/etc/profile.d/freetype.sh
-
-	# for compat. This should be removed once all apps are properly using
-	# pkg-config
-	ln -s freetype2 "$pkgdir"/usr/include/freetype
```

Comment:
I didn't verify.
I just assumed that was old cruft because it was from 2013.

How can we check to be sure?